packages/suite-desktop/src-electron/modules/auto-updater.ts

```diff
@@ -102,51 +102,56 @@ const init = ({ mainWindow, store }: Dependencies) => {
         mainWindow.webContents.send('update/checking');
     });
 
-    autoUpdater.on('update-available', updateInfo => {
-        const { version, releaseDate, files } = updateInfo;
-
+    autoUpdater.on('update-available', ({ version, releaseDate }: UpdateInfo) => {
         logger.warn('auto-updater', [
             'Update is available:',
             `- Update version: ${version}`,
             `- Release date: ${releaseDate}`,
             `- Manual check: ${b2t(isManualCheck)}`,
         ]);
 
-        latestVersion = { version, releaseDate, files };
-        mainWindow.webContents.send('update/available', { ...latestVersion, isManualCheck });
+        // files, // TODO: files not defined in types, was it used?
```

Comment:
it's weird that it's not defined in types, but it's currently not used anywhere so we could remove it completely

Reply:
[a01d8b1](https://github.com/trezor/trezor-suite/pull/4852/commits/a01d8b1e9cc6c5470631a00fc3fb6f1876d78255)